recipes/mapie/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "MAPIE" %}
+{% set version = "0.2.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0]|lower }}/{{ name|lower }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1b34c05dc9bec21997e0f25896f71c9cc11b660028e4c56e57fb49506acbc399
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - scikit-learn
+
+test:
+  imports:
+    - mapie
+    - mapie.tests
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/MAPIE/
+  summary: A scikit-learn-compatible module for estimating prediction intervals.
+  license: BSD-3-Clause
+  license_file: LICENSE
+  doc_url: https://mapie.readthedocs.io/en/latest/
+  dev_url: https://github.com/scikit-learn-contrib/MAPIE
+
+extra:
+  recipe-maintainers:
+    - gmartinonQM
+    - vtaquet
```

Comment:
@vtaquet can you confirm you are OK being a maintainer here?

Reply:
Yes, I can !